xc7/utils/prjxray_form_channels.py

```diff
@@ -929,6 +986,160 @@ def create_constant_tracks(conn):
     return create_track(vcc_node, unique_pos), \
            create_track(gnd_node, unique_pos)
 
+vpr_pkey = 999999 # This is a hack which ensures that pkeys of tiles in VPR
```

Comment:
This is super wrong

Reply:
Yes... I know that it is super wrong. This was used to ensure that pkey values never overlap in phy_tile and tile tables. Needed it for debugging.

Reply:
But there is absolutely no reason to do that.  SQLite will assign unique pkeys, and if you need to know which pkeys are remaps, add a column